modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -183,14 +183,18 @@ const renderRoute = ({route, index, depth, slug, fullyExpanded}) => {
           style={{maxHeight: getHeight(route)}}
         >
           <ul>
-            {route.entries.map((r, idx) =>
-              renderRoute({
+            {route.entries.map((r, idx) => {
```

Comment:
Nit: Not your code, but I am pretty OCD about typing out variables. `route, index`. This is likely from the old ocular code that never went through code review.